### PR TITLE
FHIRClient.getWebTarget(String baseURL) throws NPE #2450

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

**Review by JohnTimm — COMMENTED** [review]
Client.newClientBuilder() is very expensive. We should discuss some alternatives.

File: fhir-server-test/src/test/java/com/ibm/fhir/client/test/FHIRClientTestBase.java [hunk 1]
```diff
@@ -31,25 +31,32 @@
...
     @BeforeMethod
-    public void createClient() throws Exception {
+    public void createClientAuthClient() throws Exception {
         client = FHIRClientFactory.getClient(testProperties);
     }
-    
+
+    @BeforeMethod
+    public void createOauthClient() throws Exception {
+        client = FHIRClientFactory.getClient(testPropertiesOAuth2);
```
Comment (tbieste) [inline fhir-server-test/src/test/java/com/ibm/fhir/client/test/FHIRClientTestBase.java:57]: Should this be clientOAuth2?

**Review by tbieste — APPROVED** [review]
LGTM

**Review by JohnTimm — APPROVED** [review]
LGTM

**Pull request merged**